spec/Section 5 -- Validation.md

```diff
@@ -556,7 +556,7 @@ fragment safeDifferingArgs on Pet {
 ```
 
 However, the field responses must be shapes which can be merged. For example,
-scalar values must not differ. In this example, `someValue` might be a `String`
+field types must not differ. In this example, `someValue` might be a `String`
```

Comment:
Previously suggested here: https://github.com/graphql/graphql-spec/pull/957#discussion_r889675186

```suggestion
leaf types must not differ. In this example, `someValue` might be a `String`
```

Reply:
'leaf types' is just a subgroup of types, not clear, related to fields themselves. How about: 

For example, for fields of leaf types, the fields' types must not differ.